pkg/backingstore/backingstore.go

```diff
@@ -369,13 +369,35 @@ func createCommon(cmd *cobra.Command, args []string, storeType nbv1.StoreType, p
 		log.Fatalf(`❌ %s %s`, validationErr, cmd.UsageString())
 	}
 
+	suggestedSecret := util.CheckForIdenticalSecretsCreds(secret, util.MapStorTypeToMandatoryProperties[storeType])
+	if suggestedSecret != nil {
+		var decision string
+		log.Printf("Found a Secret in the system with the same credentials (%s), do you want to use it for this Backingstore? y/n", suggestedSecret.Name)
+		fmt.Scanln(&decision)
+		if strings.ToLower(decision) == "y" {
+			log.Printf("Will use %s as the Backingstoe Secret", suggestedSecret.Name)
+			err := util.SetBackingStoreSecretRef(backStore, &corev1.SecretReference{
+				Name:      suggestedSecret.Name,
+				Namespace: suggestedSecret.Namespace,
+			})
+			if err != nil {
+				log.Fatalf(`❌ %s`, err)
+			}
+		} else if strings.ToLower(decision) == "n" {
```

Comment:
I can print a more "relaxed" message with no ❌, just explaining the creation can't be proceed
@romayalon WDYT?

Reply:
my intention was that I wouldn't even provide the n option, I would just fail

Reply:
in that case, I think your suggestion is even a worse user experience, since the user has no option at all.

Reply:
Providing n as an option and then failing is not a different option as well, you make him think that he can use a new secret instead of the suggested one but then you fail...

Reply:
that's why I'm suggesting printing a more indicative message explaining that he must use that secret, the `log.Fatalf` is there just to stop the process.

Reply:
I would explain that while providing the "y/n" options and not after he already chose n